app/assets/javascripts/ace.js.coffee

```diff
@@ -1,4 +1 @@
-#= require ace/ace
-#= require ace/mode-javascript.js
-#= require ace/mode-markdown.js
-#= require ace/mode-coffee.js
+#= require ace-rails-ap
```

Comment:
Hmm..
I've just followed in according to [documentation](http://www.rubydoc.info/gems/ace-rails-ap/4.1.1).

But i can do mode/themes pluggable via .env for example:
```
ACE_EDITOR_MODES=javascript,coffee,markdown,sql,yaml
ACE_EDITOR_THEMES=sqlserver,github
```
**app/assets/javascripts/ace.js.erb:**
```ruby
<% ENV['ACE_EDITOR_MODES'].split(',').each do |mode| %>
<% require_asset "ace/mode-#{mode}.js" %>
<% end %>

<% ENV['ACE_EDITOR_THEMES'].split(',').each do |theme| %>
<% require_asset "ace/theme-#{theme}.js" %>
<% end %>
```
is this proper way?)